analysis/tests/src/expected/TypeContextCompletion_Jsx.res.txt

```diff
@@ -1,31 +1,5581 @@
-Complete src/TypeContextCompletion_Jsx.res 16:35
-posCursor:[16:35] posNoWhite:[16:34] Found expr:[16:12->16:35]
-JSX <SomeComponent:[16:12->16:25] someVaria[16:26->16:35]=...[16:26->16:35]> _children:None
+Complete src/TypeContextCompletion_Jsx.res 20:35
+posCursor:[20:35] posNoWhite:[20:34] Found expr:[20:12->20:35]
+JSX <SomeComponent:[20:12->20:25] someVaria[20:26->20:35]=...[20:26->20:35]> _children:None
 Completable: Cjsx([SomeComponent], someVaria, [someVaria])
 [{
     "label": "someVariant",
     "kind": 4,
     "tags": [],
-    "detail": "'someVariant",
+    "detail": "someVariant",
     "documentation": null
   }]
 
-Complete src/TypeContextCompletion_Jsx.res 19:38
-posCursor:[19:38] posNoWhite:[19:37] Found expr:[19:12->19:38]
-JSX <SomeComponent:[19:12->19:25] someVariant[19:26->19:37]=...__ghost__[0:-1->0:-1]> _children:None
+Complete src/TypeContextCompletion_Jsx.res 23:38
+posCursor:[23:38] posNoWhite:[23:37] Found expr:[23:12->23:38]
+JSX <SomeComponent:[23:12->23:25] someVariant[23:26->23:37]=...__ghost__[0:-1->0:-1]> _children:None
+Found typed context
+Completable: CtypedContext(JsxProp(<SomeComponent someVariant="" />))
+Found variant type someVariant
+[{
+    "label": "One",
+    "kind": 4,
+    "tags": [],
+    "detail": "One\n\n",
+    "documentation": null
+  }, {
+    "label": "Two",
+    "kind": 4,
+    "tags": [],
+    "detail": "Two\n\n",
+    "documentation": null
+  }, {
+    "label": "Three",
+    "kind": 4,
+    "tags": [],
+    "detail": "Three\n\n",
+    "documentation": null
+  }, {
+    "label": "Four",
+    "kind": 4,
+    "tags": [],
+    "detail": "Four\n\n",
+    "documentation": null
+  }, {
+    "label": "Five(_)",
+    "kind": 4,
+    "tags": [],
+    "detail": "Five(int)\n\n",
+    "documentation": null
+  }, {
+    "label": "Six(_)",
+    "kind": 4,
+    "tags": [],
+    "detail": "Six(option<string>, int)\n\n",
+    "documentation": null
+  }, {
+    "label": "someValue",
+    "kind": 12,
+    "tags": [],
+    "detail": "someVariant",
+    "documentation": null
+  }, {
+    "label": "Six",
+    "kind": 4,
+    "tags": [],
+    "detail": "Six(option<string>, int)\n\ntype someVariant =\n  | One\n  | Two\n  | Three\n  | Four\n  | Five(int)\n  | Six(option<string>, int)",
+    "documentation": null
```

Comment:
There's a bunch of duplication etc I'll need to deal with here. The top variants completed is the new work I'm doing, and the ones on the bottom comes from the general "complete from context". 

Also notice the detail is looking better on the ones completed from the general context. Not sure why, need to look into that.